vita3k/gui/src/settings_dialog.cpp

```diff
@@ -185,6 +185,13 @@ static bool get_custom_config(GuiState &gui, EmuEnvState &emuenv, const std::str
                 config.fps_hack = gpu_child.attribute("fps-hack").as_bool();
             }
 
+            // Load Audio Config
+            if (!config_child.child("audio").empty()) {
+                const auto audio_child = config_child.child("audio");
+                config.audio_volume = audio_child.attribute("audio-volume").as_float();
```

Comment:
You changed type of `audio_volume` class field to int, but here you load it as float

Reply:
😐🤦‍♂️😂
Sorry 😐🤦‍♂️

Reply:
Done 🗿